packages/types/src/promotion/common/application-method.ts

```diff
@@ -0,0 +1,31 @@
+import { BaseFilterable } from "../../dal"
+import { PromotionDTO } from "./promotion"
+
+export type ApplicationMethodType = "fixed" | "percentage"
+export type ApplicationMethodTargetType = "order" | "shipping" | "item"
+export type ApplicationMethodAllocation = "each" | "across"
+
+export interface ApplicationMethodDTO {
+  id: string
+}
+
+export interface CreateApplicationMethodDTO {
+  type: ApplicationMethodType
+  target_type: ApplicationMethodTargetType
+  allocation?: ApplicationMethodAllocation
+  value?: number
+  max_quantity?: number
+  promotion?: PromotionDTO | string
+}
+
+export interface UpdateApplicationMethodDTO {
+  id: string
```

Comment:
question: we will add the necessary properties later?

Reply:
same question for all update interfaces that only refer to id

Reply:
yes, this is just a placeholder for now, will fill them up when I get to update tickets